portal-api/src/v1/routes.js

```diff
@@ -28,25 +27,50 @@ const { ukefDecisionReport, unissuedFacilitiesReport } = require('./controllers/
 const dealImportBssEwcsController = require('./controllers/data-migration/deal-import-bss-ewcs.controller');
 const dealImportGefController = require('./controllers/data-migration/deal-import-gef.controller');
 
-const users = require('./users/routes');
-const { cleanXss, fileUpload } = require('./middleware');
+const { fileUpload } = require('./middleware');
 
+const users = require('./users/routes');
 const gef = require('./gef/routes');
 
-const authRouter = express.Router();
 const openRouter = express.Router();
+
+openRouter.route('/login').post(users.login);
+
+// 1. Request password reset
+openRouter.route('/users/reset-password').post(users.resetPassword);
```

Comment:
Should both this and the route on line 43 stay on the `openRouter`? I think these are the ones that would be used if you've forgotten your password and are resetting it, so shouldn't be locked behind being logged in.

Reply:
Agreed with you here! I'm assuming these either send an email to the registered email or similar so if that's the case it's fine for it to be open. 
For the password reset token have to look at that, not sure how and when that's called. I'll come take a look later unless someone beats me to it looking at this!

Reply:
I think both these look okay to keep open.

`reset-password` is used in a "I've forgotten my password" link. So you shouldn't need to be logged in for that.

And `reset-password/:resetPwdToken` looks like it comes from the link in a reset password email. So again that feels like you shouldn't need to be logged in for it.

Reply:
That is correct @JennyHumphries